lib/datasource/pod/index.ts

```diff
@@ -32,13 +33,15 @@ function shardParts(lookupName: string): string[] {
 
 function releasesGithubUrl(
   lookupName: string,
-  opts: { account: string; repo: string; useShard: boolean }
+  opts: { hostURL: string; account: string; repo: string; useShard: boolean }
 ): string {
-  const { useShard, account, repo } = opts;
-  const prefix = 'https://api.github.com/repos';
+  const { hostURL, account, repo, useShard } = opts;
+  const prefix = hostURL
+    ? `${hostURL}/api/v3/repos`
+    : 'https://api.github.com/repos';
   const shard = shardParts(lookupName).join('/');
-  const suffix = useShard ? `${shard}/${lookupName}` : lookupName;
-  return `${prefix}/${account}/${repo}/contents/Specs/${suffix}`;
+  const suffix = useShard ? `${repo}/${shard}/${lookupName}` : lookupName;
```

Comment:
Repo name (`Specs`) should only be appended again for shard URL only. Update the unit test and snapshot to reflect the same.

Reply:
Please provide a sample test repo, which shows this required changes.

Reply:
* [Steps](https://guides.cocoapods.org/making/private-cocoapods.html) for creating repo.
* Creating a New PodSpec Repository [article1](https://medium.com/@shahabejaz/create-and-distribute-private-libraries-with-cocoapods-5b6507b57a03) and [article2](https://medium.com/keove/creating-private-public-pods-for-cocoapods-2fc69b30270b)
* Sample of repos
   * https://github.com/keove/PodSpecs
   * https://github.com/shahabejaz/PrivatePods
* Sample of repo with shared
   * https://github.com/CocoaPods/Specs/tree/master/Specs/a/7/6/Analytics